tasmota/tasmota_xdrv_driver/xdrv_90_esp32_dingtian_relay.ino

```diff
@@ -56,7 +59,7 @@ uint32_t DingtianReadWrite(uint32_t outputs)
 
   // setup
   digitalWrite(Dingtian->pin_rck, 0);    // rclk and clkinh to 0
-  digitalWrite(Dingtian->pin_pl, 1);      // load inputs in '165, ready for shift-in (side effect '595 in tri-state)
+  if (PinUsed(GPIO_DINGTIAN_PL)) digitalWrite(Dingtian->pin_pl, 1);      // load inputs in '165, ready for shift-in (side effect '595 in tri-state)
```

Comment:
See my other comment below. PL is a mandatory pin so this test is useless.

Reply:
PL is only mandatory for old board, for new board OE is mandatory

Reply:
PL is driven via and inverter connected to RCLK.